package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vscode-kubernetes-tools-api",
-  "version": "1.0.0",
+  "version": "1.2.0",
```

Comment:
I'm not sure what your policy is for changes to the package-lock.json. Please let me know if you prefer this to be removed from this PR.